lib/public_key/src/pubkey_cert.erl

```diff
@@ -26,23 +26,23 @@
 
 -export([init_validation_state/3, 
          prepare_for_next_cert/2,
- 	 validate_time/3, 
+         validate_time/3, 
```

Comment:
I don't know what is going on with whitespaces, but now they are all spaces and not a mix between tabs and spaces.

Reply:
This is probably due to it not being enforced to be one way a long time ago.  So code base slowly becomes more adherent.